app/models/decidim/initiative.rb

```diff
@@ -80,7 +80,7 @@ class Initiative < ApplicationRecord
     scope :order_by_supports, -> { order("initiative_votes_count + coalesce(offline_votes, 0) desc") }
     scope :order_by_most_commented, lambda {
       select("decidim_initiatives.*")
-        .joins(%q(LEFT OUTER JOIN "decidim_comments_comments" ON "decidim_comments_comments"."decidim_commentable_id" = "decidim_initiatives"."id" AND "decidim_comments_comments"."decidim_commentable_type" = 'Decidim::Initiative'))
```

Comment:
is this change intended?

Reply:
Oh god, no! It should be replaced by native left outer join feature added in rails 5. I'll fix.

Reply:
I was thinking "OMG is rubocop THIS clever???", I guess it isn't 😂

Reply:
Yes, I replaced the SQL dependant  left outer join by the .left_joins that offers Rails 5.

Reply:
So the changes are correct? Can this PR be merged then?

Reply:
Yes, this can be merged.